### ISSUE TITLE
Probleme Namensgebung bei "Tot" eines creeps

### ISSUE THREAD

**Issue by Steale** [Steale/screeps#3]
Durch "Zähler" kann es vorkommen, dass ein Screep nach einem vorhandenen benannt wird

**Comment by StreamKing** [comment]
Entweder eine ausreichend große Zufallszahl benutzen oder eine HashMap zum managen von Namen verwenden.. Merkt man irgendwo wenn ein creep stirbt ?

**Comment by Steale** [comment]
ja, du kannst ja mit der funktion arbeiten, die auch die Memory von nicht vorhandenen creeps löscht. Entweder man benennt vorhanden um (was bei einer großen Anzahl schlecht ist) oder man indiziert einfach jeden screep und zählt fortlaufend hoch(extra memory Eintrag )
